src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.controller;
+
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import racingcar.dao.CarDao;
+import racingcar.dao.PlayResultDao;
+import racingcar.domain.Car;
+import racingcar.domain.RacingGame;
+import racingcar.dto.GameResultDto;
+import racingcar.dto.PlayRequestDto;
+import racingcar.view.util.TextParser;
+
+@RestController
+public class RacingCarController {
+
+    private final PlayResultDao playResultDao;
+    private final CarDao carDao;
+
+    public RacingCarController(final PlayResultDao playResultDao, final CarDao carDao) {
+        this.playResultDao = playResultDao;
+        this.carDao = carDao;
+    }
+
+    @Transactional
+    @PostMapping("/plays")
+    public ResponseEntity<GameResultDto> play(@RequestBody PlayRequestDto playRequestDto) {
+        final RacingGame racingGame = createGame(playRequestDto.getNames());
+        final int count = playRequestDto.getCount();
+
+        race(count, racingGame);
+        final List<Car> cars = racingGame.getCars();
+        final String winners = String.join(", ", racingGame.getWinnerNames());
```

Comment:
```suggestion
        final String winners = racingGame.getWinnerNames()
                .stream()
                .collect(Collectors.joining(", "));
```
간단한 로직이라 지금 상태도 좋지만 나중에 로직이 복잡해진다면 
stream의 Collectors.joining을 이용하는 방법도 있습니다 ㅎㅎ